internal/vm-console-proxy-bundle/bundle.go

```diff
@@ -142,5 +144,10 @@ func validateBundle(bundle *Bundle) error {
 	if len(missingFields) > 0 {
 		return fmt.Errorf("bundle is missing these objects: %v", missingFields)
 	}
+
+	if len(bundle.ClusterRoles) != 2 {
```

Comment:
Do we need to hardcode this check here?

Reply:
I've just removed the condition. Making the code more generic is maybe a task for a future PR.